### DISCUSSION TITLE
marker peaks track changed each time when using plotBrowserTrack

### DISCUSSION THREAD

**Discussion by jinbufan** [GreenleafLab/ArchR#2005] [Questions / Documentation]
When I use plotBrowserTrack(), I found that the track plot pattern between marker peaks will change each time when I re-plotting with the same parameters, and the reason was y ranges changed each time, which showed as 'Coverage Norm. ATAC Signal Range (0−0.16) by ReadsInTSS'. Does anyone know how to change the ranges of y-axis?  I tried to find the solution in the original code but I failed...
By the way, I tried to change 'ylim' parameter to 'c(0,0.9)' as previous discussion showed, but the actual plot's range turned to 0-0.03, I have no idea either.
I will really appreciate if someone could solve my problem. Please check the plot below for more details. 
![微信图片_20230819194415](https://github.com/GreenleafLab/ArchR/assets/142666057/911ad06b-1ca1-47d5-9944-83868b704184)

**Comment by rcorces** [comment]
You cannot specify a specific y axis limit, just a quantile limit. The tracks differ because of random sampling. Currently, the only way around this is implemented on the `dev` branch and you have to use the `maxCells` parameter and set it to a value larger than the maximum number of cells in any group. This effectively disables random sampling of cells.

**Reply by jinbufan** [reply]
Thanks for your reply! I try to set scCellsMax parameter in plotBrowserTrack() but the tracks are still differ, or should I set maxCells in addReproduciblePeakSet()? I'm sorry for this question but I really confused about it. Very thank you.

**Reply by ZennFangZhou** [reply]
did you  deal with that question? i met same condition when i repeatly plot the tarcks, i found the result became opposite...

**Reply by jinbufan** [reply]
sorry but still not, maybe use signac for plotting will be better

**Reply by ZennFangZhou** [reply]
devtools::install_github("GreenleafLab/ArchR", ref="dev", repos = BiocManager::repositories(), force=TRUE). the code can install 1.0.3 version ArchR, you can plot tracks of all cells by para maxCells.
